cluster/images/etcd/Makefile

```diff
@@ -26,8 +26,8 @@
 # Except from etcd-$(tag) and etcdctl-$(tag) binaries, we also
 # need etcd and etcdctl binaries for backward compatibility reasons.
 # That binary will be set to the last tag from $(TAGS).
-TAGS?=2.2.1 2.3.7 3.0.17
```

Comment:
If we want to support 2.2 -> 3.1 upgrade with this script, then all versions are neceesary. And we still officially support etcd v2 (which means probably 2.2.1 because we never really qualified 2.3.*).